paasta_tools/paasta_cli/utils.py

```diff
@@ -540,3 +540,45 @@ def get_jenkins_build_output_url():
     if build_output:
         build_output = build_output + 'console'
     return build_output
+
+
+class PaastaValidateMessages:
+
+    """Collection of messages printed out by 'paasta validate'.
+    Helpful as it avoids cumbersome maintenance of the unit tests.
+    """
+
+    SCHEMA_VALID = success("Successfully validated schema")
+    SCHEMA_INVALID = failure(
+        "Failed to validate schema. More info:", "http://y/yelpsoa-configs")
+
+
+class PaastaValidateSchemas:
+
+    """Collection of jsonschemas used by 'paasta validate'.
+    Helpful as it avoids cumbersome maintenance of the unit tests.
+    """
+
+    MARATHON_SCHEMA = {
```

Comment:
I understand the desire to have them in python, but please do just normal json for these json schemas. I think that will be least surprising and makes it easier to interact with other jsonschema tools.

One of the things that helped me write the first schema was using an online interactive json schema tool, and this would be a bit harder to do if it wasn't *actually* json.